src/containers/BookList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prop-types
+const BookList = ({ books }) => (
+  <div>
+    <table>
+      <tr>
+        <th> Book ID </th>
+        <th> Title </th>
+        <th> Category </th>
+      </tr>
+      {
+        // eslint-disable-next-line react/prop-types
+        books.map(book => (
+          <tr key={book.bookId}>
+            <td>{ book.bookId }</td>
+            <td>{ book.title }</td>
+            <td>{ book.category }</td>
+          </tr>
+        ))
+      }
+    </table>
+  </div>
+);
+BookList.prototype = {
+  books: PropTypes.array,
+};
```

Comment:
There's a typo with this current PropTypes definition. This is mainly the reason we having linter issues here. I would recommend we remove the linter rule that is been disabled and do the following:

- There seem to be `prototype` been used instead of `propTypes`
- Let's use `arrayOf` to define the type of array we are passing as props
- Since its an array of objects and we are using those object properties directly, let's use `shape` to defined the properties of the object. See https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes for more examples and below for how it should be.


```jsx
BookList.propTypes = {
  books: PropTypes.arrayOf(
    PropTypes.shape({
      bookId: PropTypes.string.isRequired,
      title: PropTypes.string.isRequired,
      category: PropTypes.string.isRequired,
    }).isRequired,
  ).isRequired,
};
```